### PR TITLE
Challenge/anna sala/to do full stack

### PR DESCRIPTION
**Pull request by annasm07**

### CODE REVIEW

File: anna-sala/challenges/to-do-full-stack/frontend/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Dashboard from './components/Dashboard';
+
+function App() {
+  return (
+    <>
+      <Router>
+        <h1 className="page-title">TO DO LIST</h1>
+        <Switch>
+          <Route exact path="/" component={Dashboard} />
+          <Redirect to="/" />
```
Comment (gteixi) [inline anna-sala/challenges/to-do-full-stack/frontend/src/App.jsx:18]: Why do u use redirect if you have the same path to the component?

**Review by gteixi — APPROVED** [review]
Good project with nice coverage Anna!! 👍

File: anna-sala/challenges/to-do-full-stack/frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
```
Comment (Laura-SM) [inline anna-sala/challenges/to-do-full-stack/frontend/public/index.html:28]: Remember change title!

File: anna-sala/challenges/to-do-full-stack/frontend/src/redux/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import tasksReducer from './tasksReducer';
+// import selectedTaskReducer from './selectedTaskReducer';
+
+const rootReducer = combineReducers({
+  tasks: tasksReducer,
+//   selectedTask: selectedTaskReducer,
+});
```
Comment (Laura-SM) [inline anna-sala/challenges/to-do-full-stack/frontend/src/redux/reducers/index.js:8]: Delete comments

**Review by Laura-SM — APPROVED** [review]
Good job!

**Review by IrenePerezAlonso — APPROVED** [review]

**Review by fredyzz — APPROVED** [review]
Good job despite yesterday's betrayal!

**Pull request merged**